x-pack/plugins/canvas/canvas_plugin_src/functions/common/plot/get_flot_axis_config.ts

```diff
@@ -49,10 +49,10 @@ export const getFlotAxisConfig = (
       position && acceptedPositions.includes(position) ? position : acceptedPositions[0];
 
     if (axisType === 'number' || axisType === 'date') {
-      if (min !== null) {
+      if (min != null) {
```

Comment:
What about being a bit more explicit and doing 
```js
if (min !== null && min !== undefined)
```
Would maybe help prevent this happening again.

Reply:
Surprised that the linter OKs coercive predicates. In any case, the code doesn't just want something that's not a `null` or `undefined`, looks like it wants a `number` that's also a finite one (excluding those IEEE754 values that are NaN or +/- Infinity), so we might as well use [Number.isFinite](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite#Examples)
```
  if (Number.isFinite(min)) {
```